singleimg.js

```diff
@@ -0,0 +1,27 @@
+function searchPhoto() {
+    let clientId = "LEIe9GLnkRsNK1OL9IvVZsaz-m0WWmi0-9Z4rMQFtv0";
+    let query = document.getElementById('searchMe').value;
+
+    let url = "https://api.unsplash.com/search/photos/?client_id=" + clientId + "&query=" +query;
+
+
+
+    //make request to the api
+    fetch(url)
+    .then(function(data) {
+        return data.json(); 
+    })
+    .then(function(data){
+        console.log(data);
+
+        data.results.forEach(photo  => {
```

Comment:
to get a random photo from the array of 10 pics that are got, you could use the math.floor(math.random()*array.length) to get a random index then call that one from the data.